include/logging/log_core.h

```diff
@@ -356,6 +368,11 @@ extern "C" {
 #define LOG_RUNTIME_FILTER(_filter) LOG_LEVEL_DBG
 #endif
 
+/** @brief Log level value used to indicate log entry that should not be
+ *	   formatted (raw string).
+ */
+#define LOG_LEVEL_INTERNAL_RAW_STRING ((1 << LOG_LEVEL_BITS) - 1)
```

Comment:
We have similar functionality for printk() handling. Why here we are introducing another version of the same?

Reply:
yes, normally there is a field in the message but in sync mode we don't have messages so we need to pass that information somehow. Instead of additional function flag i used level field.

Reply:
Could we use 1 scheme for both situations?

Reply:
changed to use the new scheme in both cases.